### DISCUSSION TITLE
OpenGL & Intel + AMD

### DISCUSSION THREAD

**Discussion by ZeChArtiahSaher** [italomandara/CXPatcher#176] [Q&A]
I have two questions mainly:
- possible to have something like this working ? (the goal is to have opengl 3/4 running). https://github.com/pal1000/mesa-dist-win got it from this discussion on reddit: https://www.reddit.com/r/macgaming/comments/126xvrq/running_modern_opengl_windows_games_on_crossover/
- Why is d3dmetal not switchable on intel mac with amd gfx like I don't see a reason, game porting toolkit on osx are built for x86 as well smh

**Comment by italomandara** [comment]
- I tried but it doesn't work
- d3d metal is Apple silicon only
